src/scenes/Yhteydenotto/index.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+const ContentFi = (
+  <div className="mx-auto w-full sm:w-3/4 mb-16">
+    <h1 className="mb-2">
+      Yhteydenotto
+    </h1>
+    <h2 className="mb-2">
+      Kysy tai anna palautetta
+    </h2>
+    <p className="mb-2">
+      Oiva sivustoon liittyvää palautetta ja kysymyksiä voit lähettää osoitteeseen: <a href="mailto:oivapalvelu@minedu.fi">oivapalvelu@minedu.fi</a>
+    </p>
+    <p className="mb-2">
+      Kaikki palaute luetaan ja lähetetään tarvittaessa tiedoksi asiasta vastaavalle virkamiehelle. Kysymyksiin pyrimme vastaamaan mahdollisimman pian. Vastaus lähetetään sähköpostilla ja edellyttää, että olet antanut sähköpostiosoitteesi.
+    </p>
+    <p className="mb-2">
+      Viestiin pyydetään seuraavat lähettäjän tiedot: nimi, sähköposti tai muu yhteystieto ja organisaatio.
+    </p>
+    <h2 className="mb-2">
+      Opetus- ja kulttuuriministeriön yhteystiedot
+    </h2>
+    <p className="mb-2">
+      <a href="https://minedu.fi/yhteystiedot" target="_blank">Opetus- ja kulttuuriministeriön yhteystiedot</a>
```

Comment:
Tähän ja toiseen `target="_blank"` kohtaan voisi lisätä kääntäjän vinkkaamat lisäykset:
`Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener`